set_keepalive compilation error

Between 0.7.8 and 0.7.11 this error has been introduced:

> error[E0053]: method `set_keepalive` has an incompatible type for trait
   --> .cargo/registry/src/github.com-1ecc6299db9ec823/actix-web-0.7.11/src/server/mod.rs:337:43
    |
277 |     fn set_keepalive(&mut self, dur: Option<time::Duration>) -> io::Result<()>;
    |                                      ---------------------- type in trait
...
337 |     fn set_keepalive(&mut self, _nodelay: bool) -> io::Result<()> {
    |                                           ^^^^ expected enum `std::option::Option`, found bool
    |
    = note: expected type `fn(&mut tokio_uds::UnixStream, std::option::Option<std::time::Duration>) -> std::result::Result<(), std::io::Error>`
               found type `fn(&mut tokio_uds::UnixStream, bool) -> std::result::Result<(), std::io::Error>`


I'm using

```toml
actix-web = { version="=0.7.11", features = ["uds"] }
tokio-uds = "0.2.0"
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -35,7 +35,7 @@ script:
     cargo check --features rust-tls
     cargo check --features ssl
     cargo check --features tls
-    cargo test --features="ssl,tls,rust-tls" -- --nocapture
+    cargo test --features="ssl,tls,rust-tls,uds" -- --nocapture
     fi
   - |
     if [[ "$TRAVIS_RUST_VERSION" == "nightly" ]]; then

--- a/src/server/mod.rs
+++ b/src/server/mod.rs
@@ -334,7 +334,7 @@ impl IoStream for ::tokio_uds::UnixStream {
     }
 
     #[inline]
-    fn set_keepalive(&mut self, _nodelay: bool) -> io::Result<()> {
+    fn set_keepalive(&mut self, _dur: Option<time::Duration>) -> io::Result<()> {
         Ok(())
     }
 }

